lib/readline.js

```diff
@@ -72,9 +70,12 @@ function Interface(input, output, completer, terminal) {
   this.historySize = historySize;
 
   // Check arity, 2 - for async, 1 for sync
-  this.completer = completer.length === 2 ? completer : function(v, callback) {
-    callback(null, completer(v));
-  };
+  if (typeof completer === 'function') {
+    this.completer = completer.length === 2 ? 
```

Comment:
Can you clarify what you mean by this? I'm not sure what "the way it was before" means here.